packages/bits/spec/components/dialog/dialog.atom.ts

```diff
@@ -41,4 +42,10 @@ export class DialogAtom extends Atom {
     public scrollToEnd = async (): Promise<void> => browser.executeScript<void>(`arguments[0].scrollIntoView({block: "end"})`, this.getDialog());
 
     public getDialog(): ElementFinder { return super.getElement().element(by.className("modal-dialog")); }
+
+    public getDatetimePicker(): DateTimepickerAtom {
+        return Atom.findIn(DateTimepickerAtom, this.getElement().element(by.className("date-time-picker")));
+    }
+
+    public getBody(): ElementFinder { return super.getElement().element(by.className("body")); }
```

Comment:
I don't see where this "body" class is being specified in a template. Is this method needed? If so, I don't think the method should be included in the dialog atom since the body template is controlled by the consumer, and plus they can put whatever classes they want on it.

Reply:
missed that, thanks